guide/tutorial-4-more-about-controllers-20-minutes.md

```diff
@@ -97,8 +97,8 @@ in order to be able to use the corresponding API (documented in chapter 3 of the
 
 <!-- -->
 
-> **handson**:
-Just after the include statements add a macro that defines the duration of each
+> **hands on**:
+Just after including the statements add a macro that defines the duration of each
```

Comment:
The include statement refers to the following C programming language statement:
```C
#include <some_file.h>
```
Therefore, the original version should restored.